<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentSelectionFormClass</class>
 <widget class="QDialog" name="studentSelectionFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student selection</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Filter :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditFilter"/>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidgetStudents">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column/>
       <column/>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="text">
          <string>&amp;Cancel</string>
         </property>
         <property name="icon">
          <iconset resource="../res/global.qrc">
           <normaloff>:/global/cancel22.png</normaloff>:/global/cancel22.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonOK">
         <property name="text">
          <string>&amp;OK</string>
         </property>
         <property name="icon">
          <iconset resource="../res/global.qrc">
           <normaloff>:/global/ok22.png</normaloff>:/global/ok22.png</iconset>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/global.qrc"/>
 </resources>
 <connections/>
</ui>
